README.md

```diff
@@ -141,6 +141,8 @@ We also include an experimental option to use a shared [Transformer](https://arx
 Task-specific components include logistic regression and multi-layer perceptron for classification and regression tasks, and an RNN decoder with attention for sequence transduction tasks.
 To see the full set of available params, see [config/defaults.conf](config/defaults.conf). For a list of options affecting the execution pipeline (which configuration file to use, whether to enable remote logging or tensorboard, etc.), see the arguments section in [main.py](main.py).
 
+To use the ON-LSTM sentence encoder from the ICLR 2019 paper (https://arxiv.org/abs/1810.09536) , set sent_enc as 'onlstm'. To re-rerun experiments from the paper on WSJ Language Modeling, using the configuration file [config/onlstm.conf](config/onlstm.conf)
```

Comment:
Fix wonky space before comma, add the author names for the paper, and add a period at the end.

Reply:
Edited.